notes/stories/literature/novel-various.st.txt

```diff
@@ -67,6 +67,71 @@ cryptid [The party spied a herd of supposedly extinct, prehistoric species of ma
 giant person [The party spied a more than 12-foot-tall prehistoric humanoid watching over a herd of mastodons.]
 
 
+novel: Twenty Thousand Leagues Under the Seas (1872)
+====================================================
+
+:: Title
+Twenty Thousand Leagues Under the Seas
+
+:: Date
+1872
+
+:: Description
+Twenty Thousand Leagues Under the Seas (French: Vingt mille lieues sous les
+mers) is a classic science fiction adventure novel by French writer Jules
+Verne.
+
+The novel was originally serialized from March 1869 through June 1870 in
+Pierre-Jules Hetzel's fortnightly periodical, the Magasin d'éducation et de
+récréation. A deluxe octavo edition, published by Hetzel in November 1871,
+included 111 illustrations by Alphonse de Neuville and Édouard Riou. The book
+was widely acclaimed on its release and remains so; it is regarded as one of
+the premier adventure novels and one of Verne's greatest works, along with
+Around the World in Eighty Days and Journey to the Center of the Earth. Its
+depiction of Captain Nemo's underwater ship, the Nautilus, is regarded as
+ahead of its time, since it accurately describes many features of today's
+submarines, which in the 1860s were comparatively primitive vessels.
+
+:: Authors
+Jules Verne
+
+:: References
+https://en.wikipedia.org/wiki/Twenty_Thousand_Leagues_Under_the_Seas
+
+:: Ratings
+5 <paul>
+
+:: Choice Themes
+voyage under the sea [The whole story is about a wondrous expedition under the waves Earth's world ocean in a super high-tech submarine.]
+
+:: Major Themes
+speculative submarine [A main novelty of the story is Captain Nemo's super high-tech submarine, the Nautilus.]
+human vs. captivity [The trio of Professor Aronnax, Conseil, and Ned Land refused to accept that it was their shared fate to remain on board the Nautilus permanently. Although Aronnax felt conflicted because being a prisoner on the Nautilus allowed him to conduct advanced marine research.]
+bird in a gilded cage [Professor Aronnax was reluctant to leave his underwater prison on account of the unique opportunity it afforded him to conduct advanced marine research.]
```

Comment:
I'm not sure this is the right usage. The aphorism means "a luxurious prison is still a prison (and therefore not something you want to be in)". Needs a better motivation, if it applies.

Reply:
I see what you mean. I updated the motivate here to convey how the conditions on the Nautilus were luxurious to a one-dimensional scientist, like Aronnax.